assets/js/googlesitekit/datastore/user/user-info.js

```diff
@@ -0,0 +1,212 @@
+
+		if ( user !== undefined && verified !== undefined ) {
+			return;
+		}
+
+		if ( ! global._googlesitekitUserData ) {
+			global.console.error( 'Could not load core/user info.' );
+			return;
+		}
+
+		yield actions.receiveUserInfo( { ...global._googlesitekitUserData } );
+	},
+};
+
+export const selectors = {
+	/**
+	 * Gets the user info for the logged in user.
+	 *
+	 * Returns `undefined` if the user info is not available/loaded.
+	 *
+	 * Returns an object with the shape when successful:
+	 * ```
+	 * {
+	 *   user: <Object>,
+	 *   verified: <Boolean>,
+	 * }
+	 * ```
+	 *
+	 * @private
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(Object|undefined)} User info.
+	 */
+	getUser( state ) {
+		const { user, verified } = state;
+		return { user, verified };
+	},
+
+	/**
+	 * Gets the ID for this user.
+	 *
+	 * Returns ID of the user or `undefined` if the user info is not available/loaded.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(number|undefined)} The user ID.
+	 */
+	getID: createRegistrySelector( ( select ) => () => {
+		const { user } = select( STORE_NAME ).getUser();
+		return user !== undefined ? user.id : user;
+	} ),
+
+	/**
+	 * Gets the Name for this user.
+	 *
+	 * Returns Name of the user or `undefined` if the user info is not available/loaded.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(string|undefined)} The user ID.
+	 */
+	getName: createRegistrySelector( ( select ) => () => {
+		const { user } = select( STORE_NAME ).getUser();
+		return user !== undefined ? user.name : user;
+	} ),
+
+	/**
+	 * Gets the Email for this user.
+	 *
+	 * Returns email of the user or `undefined` if the user info is not available/loaded.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(string|undefined)} The user ID.
+	 */
+	getEmail: createRegistrySelector( ( select ) => () => {
+		const { user } = select( STORE_NAME ).getUser();
+		return user !== undefined ? user.email : user;
+	} ),
+
+	/**
+	 * Gets the url to the picture for this user.
+	 *
+	 * Returns url of the user picture or `undefined` if the user info is not available/loaded.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(string|undefined)} The user ID.
+	 */
+	getPicture: createRegistrySelector( ( select ) => () => {
+		const { user } = select( STORE_NAME ).getUser();
+		return user !== undefined ? user.picture : user;
+	} ),
+
+	/**
+	 * Gets the Email for this user.
+	 *
+	 * Returns the true if the user is verified, false if not verified, or `undefined` if the user info is not available/loaded.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(boolean|undefined)} The user ID.
+	 */
+	isVerified: createRegistrySelector( ( select ) => () => {
+		const { verified } = select( STORE_NAME ).getUser() || {};
```

Comment:
This value should come directly from the datastore state instead (no need for `createRegistrySelector` here). What this will need though (based on the other comment that `getUser` should not include `verified`) is a resolver that is essentially the same as the resolver for `getUser`, just checking for the `isVerified` selector instead.